doc/classes/CodeEdit.xml

```diff
@@ -579,7 +579,8 @@
 		</member>
 		<member name="layout_direction" type="int" setter="set_layout_direction" getter="get_layout_direction" overrides="Control" enum="Control.LayoutDirection" default="2" />
 		<member name="line_folding" type="bool" setter="set_line_folding_enabled" getter="is_line_folding_enabled" default="false">
-			Sets whether line folding is allowed.
+			If [code]false[/code], line folding methods like [method fold_line] will not work and [method can_fold_line] will always return [code]false[/code].
+			If [code]true[/code], lines can be folded. See [member gutters_draw_fold_gutter].
```

Comment:
I have it this way since false is default, but I can switch it if wanted.

Reply:
It's not a deal breaker. The current description **is** pretty good and the splitting looks nice. Perhaps it could be like this, instead:
```suggestion
			If [code]true[/code], lines can be folded. Otherwise, line folding methods like [method fold_line] will not work and [method can_fold_line] will always return [code]false[/code]. See [member gutters_draw_fold_gutter].
```